packages/@webex/internal-plugin-conversation/src/encryption-transforms.js

```diff
@@ -82,6 +82,14 @@ export const transforms = toArray('outbound', {
   },
 
   encryptActivity(ctx, key, activity) {
+    // Meeting Container policy update requests do not need to be encrypted.
```

Comment:
Yes, these changes will only impact a very narrow scope of requests with a target type of `meetingContainer` and an object type of `policies`. I have sent you a screenshot of this working within Webex to prevent personal data from leaking within the request data.